CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.0.4] - 2024-11-12
+
+### Added
+
+- `ResponseStarted`: Indicates the start of TTS audio streaming from the server.
+  The message contains the textual content of the utterance to be spoken.
+- `ResponseInterrupted`: Indicates an interruption in the TTS audio stream from the server.
+  The message contains the textual content up to the point where the utterance was stopped.
+- `ResponseCompleted`: Indicates the completion of TTS audio transmission from the server.
+- `ConversationEnding`: Indicates the session will continue in one-sided mode during TTS playback of the final words.
+  The message includes the textual content of the utterance just spoken.
```

Comment:
This line belongs with `ResponseCompleted` not `ConversationEnding`.

Reply:
Good catch! Will fix it in this PR https://github.com/speechmatics/speechmatics-flow/pull/5